### DISCUSSION TITLE
Nowy task dotyczący designu do dodatkowych uwag

### DISCUSSION THREAD

**Discussion by mattyl006** [emkarcinos/WMIAdventure#386] [Sprint planning]
Zrobiłem taska, aby zrobić w figmie jak ma wyglądać te pisanie przez użytkownika dodatkowych uwag do wysyłki karty #385.
Bo chyba zabrakło takiego taska.

**Comment by itchystone** [comment]
o racja

**Comment by emkarcinos** [comment]
Taki task już był stworzony (#353).

Usuwam.

**Reply by mattyl006** [reply]
A myślałem że ten task dotyczy popupa po wysłaniu karty
